### ISSUE TITLE
gardenctl ssh <gcp_node> fails with `Cmd was unsuccessful`

### ISSUE THREAD

**Issue by dansible** [gardener/gardenctl#443]
**Describe the bug**
`gardenctl ssh <gcp_node>` fails with `Cmd was unsuccessful` (using `gardenctl/gctl-gcp` cluster)

**To Reproduce**
Using 52235f2f with a test cluster gives:

```
$ go run cmd/gardenctl/main.go ssh shoot--gardenctl--gctl-gcp-gctl-gcp-z1-bb4d4-flzns

Warning:
Be aware that you are entering an untrusted environment!
Do not enter credentials or sensitive data within the ssh session that cluster owners should not have access to.

Downloaded id_rsa key
Check Public IP

(1/4) Fetching data from target shoot cluster
Cmd was unsuccessful
exit status 2
```

**Gardenctl Version (please complete the following information):**
 - Version/Commit 52235f2f

**Comment by neo-liang-sap** [comment]
seems working from my side - the only thing i need to do is connecting to PHILA VPN server as shanghai vpn server can't connect to google services....

![image](https://user-images.githubusercontent.com/42594392/99207028-dfcb9a80-27f7-11eb-86b7-6eff209165e0.png)
![image](https://user-images.githubusercontent.com/42594392/99207131-1bfefb00-27f8-11eb-95da-5042afc932b8.png)

**Comment by dansible** [comment]
Unfortunately i'm still not able to have an ssh session succeed on GCP. Both on or off the Philiddelphia VPN, I get this:

```
$ go run cmd/gardenctl/main.go ssh shoot--gardenctl--gctl-gcp-gctl-gcp-z1-bb4d4-xkxp6

Warning:
Be aware that you are entering an untrusted environment!
Do not enter credentials or sensitive data within the ssh session that cluster owners should not have access to.

Downloaded id_rsa key
Check Public IP

(1/4) Fetching data from target shoot cluster
Cmd was unsuccessful
exit status 2
```

Debug mode (`-d`) does not provide any more info. This is the same with the released `gardenctl` (`v0.23.1`)

**Comment by dansible** [comment]
Thanks @neo-liang-sap, I reinstalled my gcloud command line and was able to resolve the issue :+1:
